packages/validator/src/services/prepareBeaconProposer.ts

```diff
@@ -37,13 +38,19 @@ export function pollPrepareBeaconProposer(
     const indicesChunks = batchItems(validatorStore.getAllLocalIndices(), {batchSize: REGISTRATION_CHUNK_SIZE});
 
     for (const indices of indicesChunks) {
+      const proposers: routes.validator.ProposerPreparationData[] = [];
       try {
-        const proposers = indices.map(
-          (index): routes.validator.ProposerPreparationData => ({
+        indices.forEach((index) => {
+          const feeRecipient = validatorStore.getFeeRecipientByIndex(index);
+          const pubKey = validatorStore.getPubkeyOfIndex(index);
+
+          proposers.push({
             validatorIndex: String(index as number),
-            feeRecipient: validatorStore.getFeeRecipientByIndex(index),
-          })
-        );
+            feeRecipient,
+          });
+
+          logger.info("Validator exists in beacon chain", {validatorIndex: index, feeRecipient, pubKey});
```

Comment:
Can you revert the changes in this file?
There's already the other "Validator exists in beacon chain" log

Reply:
Given this includes the `feeReceipient` as well, perhaps we should remove the [_debug log_](https://github.com/ChainSafe/lodestar/pull/5161/files/50825d16cc85a747ce41c21f1f1a990fd799112d#diff-eb9b1b96c2b62069854c2962a24c1983f831836c63455470fe0d0d78c8f5812cR138-R141) in favour of this _info log_

Reply:
printing out the free recipient would be great, at the moment I always have to query the keymanager api to check this

Reply:
This is not the right place to add this log. This will get run every epoch XD
And its where we're preparing for epoch proposers.

Really recommend reverting all changes to this file and attempting the log in the other file. Really even `fetchValidatorIndices` isn't probably the best place for it, but at least it will have the right behavior (print once upon discovery)